### PR TITLE
Add tests for metrics recorder

### LINKED ISSUE #88038: Scheduler: Add tests for metrics recorder in framework/v1alpha1
**What would you like to be added**:
Tests for `metrics_recorder.go`.

/sig scheduling
/help
/kind cleanup
/assign

### PR DESCRIPTION
**Pull request by juchaosong**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #88038

**Does this PR introduce a user-facing change?**:
Add tests for metrics recorder

```release-note
NONE
```

### CODE REVIEW

File: pkg/scheduler/framework/v1alpha1/metrics_recorder_test.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+			collectAndComparePluginDurationMetrics(t, tt.extensionPoint, tt.pluginName, tt.wantStatus, tt.wantCount, tt.wantSum)
+		})
+	}
+}
+
+func collectAndComparePluginDurationMetrics(t *testing.T, wantExtensionPoint, wantPlugin string, wantStatus Code, wantCount uint64, wantSum float64) {
+	t.Helper()
+	m := collectHistogramMetric(metrics.PluginExecutionDuration)
+	if len(m.Label) != 3 {
+		t.Fatalf("Unexpected number of label pairs, got: %v, want: 2", len(m.Label))
```
Comment (php-coder) [inline pkg/scheduler/framework/v1alpha1/metrics_recorder_test.go]:
```suggestion
		t.Fatalf("Unexpected number of label pairs, got: %v, want: 3", len(m.Label))
```

Comment (sjc960115) [reply pkg/scheduler/framework/v1alpha1/metrics_recorder_test.go]: Thank you. I've fixed it.

**Pull request closed without merging**